homeassistant/components/compensation/config_flow.py

```diff
@@ -0,0 +1,142 @@
+"""Config flow for statistics."""
+
+from __future__ import annotations
+
+from collections.abc import Mapping
+from typing import Any, cast
+
+import voluptuous as vol
+
+from homeassistant.const import (
+    CONF_ATTRIBUTE,
+    CONF_ENTITY_ID,
+    CONF_NAME,
+    CONF_UNIT_OF_MEASUREMENT,
+)
+from homeassistant.helpers.schema_config_entry_flow import (
+    SchemaCommonFlowHandler,
+    SchemaConfigFlowHandler,
+    SchemaFlowError,
+    SchemaFlowFormStep,
+)
+from homeassistant.helpers.selector import (
+    AttributeSelector,
+    AttributeSelectorConfig,
+    BooleanSelector,
+    EntitySelector,
+    NumberSelector,
+    NumberSelectorConfig,
+    NumberSelectorMode,
+    ObjectSelector,
+    ObjectSelectorConfig,
+    ObjectSelectorField,
+    TextSelector,
+)
+
+from .const import (
+    CONF_DATAPOINTS,
+    CONF_DEGREE,
+    CONF_LOWER_LIMIT,
+    CONF_PRECISION,
+    CONF_UPPER_LIMIT,
+    DEFAULT_DEGREE,
+    DEFAULT_NAME,
+    DEFAULT_PRECISION,
+    DOMAIN,
+)
+
+
+async def get_options_schema(handler: SchemaCommonFlowHandler) -> vol.Schema:
+    """Get options schema."""
+    entity_id = handler.options[CONF_ENTITY_ID]
+
+    return vol.Schema(
+        {
+            vol.Required(CONF_DATAPOINTS): ObjectSelector(
+                ObjectSelectorConfig(
+                    label_field="uncompensated_value",
+                    description_field="compensated_value",
+                    multiple=True,
+                    translation_key=CONF_DATAPOINTS,
+                    fields={
+                        "uncompensated_value": ObjectSelectorField(
+                            required=True,
+                            selector=NumberSelector(
+                                NumberSelectorConfig(mode=NumberSelectorMode.BOX)
+                            ),
+                        ),
+                        "compensated_value": ObjectSelectorField(
+                            required=True,
+                            selector=NumberSelector(
+                                NumberSelectorConfig(mode=NumberSelectorMode.BOX)
+                            ),
+                        ),
+                    },
+                )
```

Comment:
When starting the flow, there's an empty data point already there:
<img width="573" height="156" alt="image" src="https://github.com/user-attachments/assets/f5fd1101-104f-48be-b208-c011130a6465" />

Reply:
Has to be a frontend bug because it's only when you configure it with multiple